.travis.yml

```diff
@@ -54,15 +54,23 @@ matrix:
 
 before_install:
   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp; fi;
-  - composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - composer global require phpunit/phpunit:@stable --no-update
-  - sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+  - |
+      if [ "$TARGET" = 'test' ]; then
+      composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN
+      composer global require phpunit/phpunit:@stable --no-update
```

Comment:
I thought about that, but then it will get very repetitive and ugly. Let's see what the log looks like, shall we?